frontends/qt/main.cpp

```diff
@@ -165,8 +165,9 @@ class WebEngineView : public QWebEngineView {
 
 int main(int argc, char *argv[])
 {
-// Enable auto HiDPI scaling on Windows only for now.
-#if defined(_WIN32) && QT_VERSION >= QT_VERSION_CHECK(5,6,0)
+// Enable auto HiDPI scaling if Qt>5.14. Otherwise do it on Windows only because it doesn't
+// look nice on Linux and MacOS.
+#if QT_VERSION >= QT_VERSION_CHECK(5,14,0) || (defined(_WIN32) && QT_VERSION >= QT_VERSION_CHECK(5,6,0))
```

Comment:
Not sure if it is worth making a distinction between Qt versions. wdyt?

Reply:
You can drop all these checks as we are on Qt 5.15 at the moment and don't support compiling this app on older Qt versions.